### DISCUSSION TITLE
S/MIME support for native macOS/iOS decryption

### DISCUSSION THREAD

**Discussion by josh1703658784** [anonaddy/anonaddy#196] [Feature Requests]
I understand GPG is preferred for many reasons but S/MIME support would be greatly appreciated.

It's not too bad on macOS to set up but it does require installing the app GPG Keychain and paying(?) for the native Mail plugin.

On iOS you need to install an alternative Mail app. 

Using GPG isn't impossible; it does require more work for my use case though. In general, I prefer to use native applications provided by the operating system. Currently I feed unencrypted data from Anonaddy to <email_provider>. <email_provider> encrypts my mail for me, but that requires me to trust AnonAddy *and* my provider. I'd rather trust only one service with plaintext info.

If there are any questions or clarification needed I would be happy to assist and answer questions.

**Comment by gitrahab** [comment]
S/MIME support would be a game charger and major selling point to advanced and casual users alike.

**Comment by willbrowningme** [comment]
Thanks for the suggestion, I should be able to add this and will look into it soon.

**Comment by gitrahab** [comment]
Cool thanks for responding on this. 

One idea I've been pondering... I imagine it would be a challenge, but would nonetheless be legendary. This would apply to both S/MIME and OpenPGP. 

Emails encrypted before forwarding to inbox. (already works)
Response to Anonaddy encrypted.
Anonaddy decrypts and send on to recipient. 

This would offer complete encryption from all 3rd party email providers (assuming reply-to address and subject lines/headers are anonymized).

Anyway I appreciate what you do and look forward to signing up for the Pro plan when S/MIME becomes available.

**Comment by josh1703658784** [comment]
Hi @willbrowningme. Is there any timeline on this feature? Thanks for all your hard work!

**Reply by willbrowningme** [reply]
Do you have an S/MIME certificate? If so, would you be able to [email it to me](https://addy.io/contact/) so I can decide on the best place for recipient's certs to be stored.

**Reply by josh1703658784** [reply]
Just shot you that over. Thanks!

**Comment by josh1703658784** [comment]
Hey @willbrowningme - wanted to bump this and see if it was still under consideration. This would open up more providers for me that I don’t wholly trust with unencrypted contents. Thanks!

**Reply by krmkrx** [reply]
Bumping this, please add this.

**Reply by willbrowningme** [reply]
Still planned, just had a lot of other stuff on my todo list!
